packages/frontend/components/Layout.tsx

```diff
@@ -1,4 +1,6 @@
 import cn from "classnames";
+import Footer, { FooterLink } from "./Footer.tsx";
```

Comment:
Export the Footer component in `helpers/components.ts` and add the FooterLink type to utils/types.ts see Header for reference